ui/pages/confirmations/components/simulation-details/useBalanceChanges.ts

```diff
@@ -20,11 +20,10 @@ import {
   Amount,
   BalanceChange,
   FIAT_UNAVAILABLE,
-  NativeAssetIdentifier,
+  NATIVE_ASSET_IDENTIFIER,
```

Comment:
Unrelated to this PR, but can you un-skip the simulation E2E tests in this PR so we can verify all our simulations changes going forward?

Reply:
Done.